samples/helloworld/gateway-api/README.md

```diff
@@ -0,0 +1,73 @@
+# Configure helloworld using the Kubernetes Gateway API
+
+Istio intends to [replace its traffic management API](https://istio.io/latest/blog/2022/gateway-api-beta/) with the Kubernetes [Gateway API](https://gateway-api.sigs.k8s.io/) as the default API for traffic management in the future.
```

Comment:
nit: "replace ... as the default" is accurate, but just slightly worried that since "replace" stands alone it may cause FUD we are going to remove the old ones?